pkgs/development/python-modules/cairocffi/default.nix

```diff
@@ -60,19 +53,19 @@ buildPythonPackage rec {
     fontDirectories = [ freefont_ttf ];
   };
 
-  propagatedNativeBuildInputs = [ cffi ];
+  propagatedNativeBuildInputs = [ cffi flit-core ];
```

Comment:
I can't wrap my head around what `propagatedNativeBuildInputs` is propagating, but I feel like the build backend a package chooses should not be propagated. Can that go in `nativeBuildInputs` instead?